controllers/yawol-controller/loadbalancer/loadbalancer_controller_test.go

```diff
@@ -38,6 +39,106 @@ const (
 	interval = time.Millisecond * 250
 )
 
+var _ = Describe("check controller-runtime predicate", func() {
+	lbSet := yawolv1beta1.LoadBalancerSet{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "lbs",
+			Namespace: "default",
+		},
+		Spec: yawolv1beta1.LoadBalancerSetSpec{
+			Selector: metav1.LabelSelector{},
+			Replicas: 1,
+			Template: yawolv1beta1.LoadBalancerMachineTemplateSpec{
+				Labels: nil,
+				Spec: yawolv1beta1.LoadBalancerMachineSpec{
+					Infrastructure:  yawolv1beta1.LoadBalancerInfrastructure{},
+					PortID:          "Port",
+					ServerGroupID:   "ServerGroup",
+					LoadBalancerRef: yawolv1beta1.LoadBalancerRef{},
+				},
+			},
+		},
+		Status: yawolv1beta1.LoadBalancerSetStatus{
+			Conditions: []metav1.Condition{
+				{
+					Type:               helper.HasKeepalivedMaster,
+					Status:             metav1.ConditionTrue,
+					LastTransitionTime: metav1.Now(),
+					Reason:             "Reason",
+					Message:            "Message",
+				},
+			},
+			ReadyReplicas: pointer.Int(1),
+			Replicas:      pointer.Int(1),
+		},
+	}
+
+	It("should reconcile", func() {
```

Comment:
`It`s should be concise and test one specific case. Can you structure it similar to what is done here: ?
https://github.com/gardener/gardener/blob/d48762a67451bb84b8fd21314ef2051eb815880e/pkg/resourcemanager/controller/tokenrequestor/add_test.go